gifc/gifc.py

```diff
@@ -0,0 +1,478 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Interact with GitHub Gists using HTTP API
+
+# python3 -m gifc -h
+# ./gifc -h
+
+
+import os
+import subprocess
+import tempfile
+import yaml
+
+import gifc.arrowshiv as arrowshiv
+arrow = arrowshiv.importArrow()
+
+
+##############################################################################
+##############################################################################
+##############################################################################
+# gifc
+##############################################################################
+
+DEBUG = False
+ERROR_SUFFIX = "Please try using the Web GUI."
+TERM = os.environ.get('TERM', '')
+
+ENABLE_COLOR = False
+if '256color' in TERM.split('-'):
+    ENABLE_COLOR = True
+
+##############################################################################
+#  Config / Environment
+##############################################################################
+
+try:
+    EDITOR = os.environ.get('EDITOR', None)  # Default editor
+    FALLBACK_EDITOR = 'nano'  # ;_; wish it was vim <3
+    GH_USER = os.environ.get('GH_USER', None)
+    GH_TOKEN = os.environ.get('GH_TOKEN', None)
+    GH_BASE_URL = "https://api.github.com"
+    HOME_DIR = os.path.expanduser("~")
+    BINARY_DIR = os.path.dirname(os.path.realpath(__file__))
+    WORKING_DIR = os.getcwd(),
+except Exception as error:
+    emsg = "FATAL Error during initialization: %s" % (error)
+    print(emsg)
+    exit(emsg)
+
+# These are shared with __main__.py
+args = []
+argvars = {}
+session = {}
+
+##############################################################################
+# Helpers - Output
+##############################################################################
+
+
+def term_width():
+    ''' Determine Terminal width for output purposes '''
+    width = 80
+    try:
+        width = os.get_terminal_size()[0]
```

Comment:
I would want you to use `os.get_terminal_size(fd=1)` explicitly. (file descriptors 0, 1 and 2: stdin, stdout and stderr)
```suggestion
        width = os.get_terminal_size(fd=1)[0]
```

Reply:
The terminal width should be the same for all, but... sure you're not wrong.  My concern would be that everything was tested without the file descriptor being selected, but doesn't see like a hard thing to test :D

Reply:
Yeah, I have already tested it and the thing is by default one of those file descriptors is being sent anyways. I just wanted to make it **explicit** :wink: